_data/header.yml

```diff
@@ -3,12 +3,21 @@
 # Uncomment this statement (by removing the leading '# ') to set the
 # header title differently from the site's title (as defined in
 # _config.yml).
-# title: Header title
+
+title: ""
```

Comment:
this doesn't quite match with the comment above. Should it be commented out, or an empty string?